AssessmentA1/TaskManager/TaskScheduler.cs

```diff
@@ -0,0 +1,38 @@
+﻿using System.Xml.Linq;
+
+namespace TaskManager
+{
+    public class TaskScheduler
+    { 
+        public IEnumerable<(Employee, TaskDetails)> ScheduleTask()
```

Comment:
Dont return tuple from a public method. Please update this method.

Reply:
Return type is changed.